ext/intl/uchar/uchar.stub.php

```diff
@@ -18,13 +18,13 @@ public static function charDigitValue(int|string $codepoint) {}
     public static function charDirection(int|string $codepoint) {}
 
     /** @return int|null */
-    public static function charFromName(string $characterName, int $nameChoice = IntlChar::UNICODE_CHAR_NAME) {}
+    public static function charFromName(string $name, int $type = IntlChar::UNICODE_CHAR_NAME) {}
```

Comment:
I'm not very sure about `$type`. Maybe `$nameType` would be better? Or let's just stay with the original name?

Reply:
I think $type is okay here.